<launch>
  <arg name="serialport"  default="/dev/ttyUSB0" />

  <node pkg="aimibot" type="aimibot" name="aimibot" output="screen">
    <param name="serial_port"     value="$(arg serialport)"/>
    <param name="odom_frame"      value="odom"/>
    <param name="base_frame"      value="base_footprint"/>
    <param name="use_imu_heading" value="true" />
    <param name="publish_tf"      value="true" />
  </node>

</launch>
